includes/controller/guest_login_controller.php

```diff
@@ -44,7 +46,7 @@ function guest_register() {
 
   $welcome_message = WelcomeMessage();
   $display_message = $welcome_message[0]['display_msg'];
-  $angel_types_source = select_angeltype();
+  $angel_types_source = select_angeltypes();
```

Comment:
A typo, as the function that exist in the system has the name 'select_angeltypes()'